main.go

```diff
@@ -13,9 +13,12 @@ import (
 	"github.com/NBISweden/sda-cli/helpers"
 	"github.com/NBISweden/sda-cli/list"
 	"github.com/NBISweden/sda-cli/upload"
+	"github.com/NBISweden/sda-cli/version"
 	log "github.com/sirupsen/logrus"
 )
 
+var Version = "development"
```

Comment:
Should the value of `version` be configurable?

Reply:
yes, the goreleaser will be able to pick up the git tag we create every time we release a new version. then it overwrites this value with the GORELEASER_CURRENT_TAG.
This is like a default value in case someone creates something like an unofficial build.